src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompositeTagHelperDescriptorResolver.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Compilation.TagHelpers;
+using Microsoft.AspNetCore.Razor.Runtime.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Internal
+{
+    public class CompositeTagHelperDescriptorResolver : ITagHelperDescriptorResolver
+    {
+        public IList<TagHelperDescriptorResolver> _resolvers;
+
+        public CompositeTagHelperDescriptorResolver(
```

Comment:
The normal `CompositeTagHelperDescriptorResolver` shouldn't have a contract for the `TagHelperDescriptorResolver` + `ViewComponentTagHelperDescriptorResolver`. We should have a `DefaultCompositeTagHelperDescriptorResolver` that has that contract.

By doing this you'll make testing easier for people extending MVC; will also enable users to more easily add their own `CompositeTagHelperDescriptorResolvers` that do not require the existing TagHelper resolution stack.